zigpy_cli/radio.py

```diff
@@ -25,9 +25,10 @@
 @click.argument("radio", type=click.Choice(list(RADIO_TO_PACKAGE.keys())))
 @click.argument("port", type=str)
 @click.option("--baudrate", type=int, default=None)
+@click.option("--flow-control", type=click.Choice(["software", "hardware", "none"]), default=None)
```

Comment:
I think `default="none"` would be better.

Zigpy expects `None` so you can convert the string where it's used into `None` if it is `"none"`.

Reply:
I had "none" there before, but then just got ugly stacktraces when removing the option together. So you are saying `if flow_control == "none" then 'None'`? I'll try if that works, that would resolve the issue I mentioned indeed.